src/views/ScopesetExpander/index.jsx

```diff
@@ -0,0 +1,123 @@
+import { hot } from 'react-hot-loader';
+import { Component, Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import { Query } from 'react-apollo';
+import { withStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import Button from '@material-ui/core/Button';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ArrowExpandVerticalIcon from 'mdi-react/ArrowExpandVerticalIcon';
+import LinkIcon from 'mdi-react/LinkIcon';
+import Dashboard from '../../components/Dashboard/index';
+import Spinner from '../../components/Spinner/index';
+import ErrorPanel from '../../components/ErrorPanel/index';
+import CodeEditor from '../../components/CodeEditor/index';
+import scopesetQuery from './scopeset.graphql';
+
+@hot(module)
+@withStyles(theme => ({
+  actionButton: {
+    ...theme.mixins.fab,
+  },
+  editor: {
+    marginBottom: theme.spacing.double,
+  },
+  title: {
+    marginBottom: theme.spacing.double,
+  },
+  listItemButton: {
+    ...theme.mixins.listItemButton,
+  },
+}))
+export default class ScopesetExpander extends Component {
+  state = {
+    scopeText: '',
+  };
+
+  handleExpandScopesClick = async () => {
+    const scopes = this.parseScopes(this.state.scopeText);
+
+    this.setState({ scopes });
+  };
+
+  handleScopesChange = scopeText => {
+    this.setState({ scopeText });
+  };
+
+  parseScopes(scopeText) {
+    return [
+      ...new Set(
+        scopeText
+          .split(/[\r\n]+/)
+          .map(s => s.trim())
+          .filter(Boolean)
+      ),
+    ];
+  }
+
+  render() {
+    const { classes, user, onSignIn, onSignOut } = this.props;
+    const { scopes, scopeText } = this.state;
+
+    return (
+      <Dashboard
+        title="Scopeset Expander"
```

Comment:
Rename this to "Expand Scopesets". We want to name these in action style rather than tool style.